### PR TITLE
Update CHANGELOG.md for 2020.06.0 release

### PR DESCRIPTION
**Pull request by krzentner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,137 @@ All notable changes to this project will be documented in this file.
...
+    [#1231](https://github.com/rlworkgroup/garage/pull/1231))
+  * PEARL in PyTorch ([#1059](https://github.com/rlworkgroup/garage/pull/1059),
+    [#1124](https://github.com/rlworkgroup/garage/pull/1124),
+    [#1218](https://github.com/rlworkgroup/garage/pull/1218),
+    [#1316](https://github.com/rlworkgroup/garage/pull/1316),
+    [#1374](https://github.com/rlworkgroup/garage/pull/1374))
+  * SAC in PyTorch ([#1235](https://github.com/rlworkgroup/garage/pull/1235))
+  * MTSAC in PyTorch ([#1332](https://github.com/rlworkgroup/garage/pull/1332))
+  * Task Embeddings in TensorFlow
+    Add TensorFlow embedding models ([#1168](https://github.com/rlworkgroup/garage/pull/1168))
```
Comment (ryanjulian) [inline CHANGELOG.md:33]: i think rendering fails here?

Comment (krzentner) [reply CHANGELOG.md]: Oh, I just forgot to update that bit. Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,137 @@ All notable changes to this project will be documented in this file.
...
+  * LocalSampler, which uses the main process to sample ([#1133](https://github.com/rlworkgroup/garage/pull/1133))
+  * MultiprocessingSampler ([#1298](https://github.com/rlworkgroup/garage/pull/1298))
+  * VecWorker, a replacement for VecEnvExecutor ([#1311](https://github.com/rlworkgroup/garage/pull/1311))
+- APIs
+  * TrajectoryBatch data type (
+    [#1058](https://github.com/rlworkgroup/garage/pull/1058),
+    [#1065](https://github.com/rlworkgroup/garage/pull/1065))
+  * TimeStep data type ([#1114](https://github.com/rlworkgroup/garage/pull/1114))
+  * TimeStepBatch data type ([#1529](https://github.com/rlworkgroup/garage/pull/1529))
+  * evaluation interface ([#1116](https://github.com/rlworkgroup/garage/pull/1116))
```
Comment (ryanjulian) [inline CHANGELOG.md:46]: Evaluaiton?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,137 @@ All notable changes to this project will be documented in this file.
...
+    [#1218](https://github.com/rlworkgroup/garage/pull/1218),
+    [#1316](https://github.com/rlworkgroup/garage/pull/1316),
+    [#1374](https://github.com/rlworkgroup/garage/pull/1374))
+  * SAC in PyTorch ([#1235](https://github.com/rlworkgroup/garage/pull/1235))
+  * MTSAC in PyTorch ([#1332](https://github.com/rlworkgroup/garage/pull/1332))
+  * Task Embeddings in TensorFlow (
+    [#1168](https://github.com/rlworkgroup/garage/pull/1168),
+    [#1169](https://github.com/rlworkgroup/garage/pull/1169),
+    [#1167](https://github.com/rlworkgroup/garage/pull/1167))
+- Samplers
```
Comment (ryanjulian) [inline CHANGELOG.md:36]: you should probably mention a rewritten Sampler API, or did that land in 2019.10?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,137 @@ All notable changes to this project will be documented in this file.
...
+  [#1414](https://github.com/rlworkgroup/garage/pull/1414),
+  [#1432](https://github.com/rlworkgroup/garage/pull/1432))
+
+### Removed
+- `RLAlgorithm.get_itr_snapshot` ([#1054](https://github.com/rlworkgroup/garage/pull/1054))
+- Dependencies:
+  * matplotlib (moved to dev) ([#1083](https://github.com/rlworkgroup/garage/pull/1083))
+  * atari-py ([#1194](https://github.com/rlworkgroup/garage/pull/1194))
+  * gtimer, pandas, rlkit, seaborn ([#1325](https://github.com/rlworkgroup/garage/pull/1325))
+- nb_utils.py ([#1288](https://github.com/rlworkgroup/garage/pull/1288))
```
Comment (ryanjulian) [inline CHANGELOG.md:119]: garage.misc.nb_utils

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,137 @@ All notable changes to this project will be documented in this file.
...
+- garage.np.regressors ([#1493](https://github.com/rlworkgroup/garage/pull/1493))
+- np/batch_polopt.py ([#1492](https://github.com/rlworkgroup/garage/pull/1492))
```
Comment (ryanjulian) [inline CHANGELOG.md:121]: garage.np.BatchPolopt

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,137 @@ All notable changes to this project will be documented in this file.
...
+- garage.misc.prog_bar_counter and pyprind ([#1495](https://github.com/rlworkgroup/garage/pull/1495))
+- garage.tf.envs.TfEnv ([#1443](https://github.com/rlworkgroup/garage/pull/1443))
+- TensorFlow BatchPolopt ([#1504](https://github.com/rlworkgroup/garage/pull/1504))
```
Comment (ryanjulian) [inline CHANGELOG.md:124]: garage.tf.BatchPolopt

**Comment by ryanjulian** [comment]
Some things to consider adding, based on previous CHANGELOG:

* List of dependency upgrades or relaxations, if any (e.g. you can now use TF2 if you want)
* Any packaging changes (new extras, removal of extras e.g. `gpu`)

File: CHANGELOG.md (hunk 3 above)
Comment (krzentner) [reply CHANGELOG.md:43]: Either way, it's not mentioned in the 2019.10 changes.

**Comment by ryanjulian** [comment]
@Mergifyio rebase

**Pull request merged**